src/modules/Checkbox/Checkbox.js

```diff
@@ -149,6 +149,10 @@ export default class Checkbox extends Component {
     }
   }
 
+  handleMouseDown = (e) => {
+    _.invoke('focus', this.checkboxRef)
```

Comment:
This is correct, however, we also need to call the user's `onMouseDown` prop here if they provide one.  That is because we handling the prop, so it isn't auto spread on the rendered component.  Even if it were, we are also setting the onMouseDown event ourselves, so our handler is the only one that will ever be called.

So, if they pass:
```jsx
<Checkbox onMouseDown={() => console.log('hi')} />
```
Then nothing will happen onMouseDown.  To ensure they can also pass a `onMouseDown` handler, we need to also call it in our handler for them.   Since mouse down is called before the focus event, let's call it in the right order as well:

```jsx
handleMouseDown = (e) => {
  const { onMouseDown } = this.props

  if (onMouseDown) onMouseDown(e)

  _.invoke('focus', this.checkboxRef)
}
```